package.json

```
@@ -30,7 +30,7 @@
   },
   "homepage": "https://github.com/JuStTheDev/stylelint-magic-numbers#readme",
   "peerDependencies": {
-    "stylelint": "^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0"
+    "stylelint": ">=^9.0.0"
```

Hi and thanks for your contribution!
Wouldn't it then make even more sense to remove the "^" completely? 
```suggestion
    "stylelint": ">=9.0.0"
```